Assets/Scripts/TriggeredTorch.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TriggeredTorch : MonoBehaviour
+{
+ 
+   [SerializeField]
+    private GameObject Torch;
+    private Collider2D fireBall;
+    private Collider2D onTorchCol;
+    public Sprite onTorch; 
+
+     void Start()
+    {
+        fireBall = GameObject.Find("Fireball").GetComponent<Collider2D>();
+        onTorchCol = GameObject.Find("Torch").GetComponent<Collider2D>();
+    }
+    void Awake()
+    {
+        Torch.SetActive(false);   // ensuring the torch is off by default
```

Comment:
can you let level designers choose the starting state of the torch with a public/Serialized bool?